test/test_data/SigningUtilMock.ts

```diff
@@ -0,0 +1,23 @@
+import { Presentation } from '../../lib';
+
+export class SigningUtilMock {
+  public getSinged(): Presentation[] {
```

Comment:
Hmzz, not entirely sure I like a mock that looks nothing like an actual real thing here. Can we at least do something with the provided callback arguments to test it. Because right now because of the callback being passed in the tests are literally testing that a callback mechanism works in TS. Of course if I pass in the function and then check against actually calling that function the result will be what you would expect. Can we at least extend it to see that if a callback conforms to the signature that all arguments are being handled for instance

Reply:
improved a little.